selfdrive/car/toyota/values.py

```diff
@@ -24,51 +24,82 @@ class ToyotaFlags(IntFlag):
   HYBRID = 1
 
 
-class CAR:
+class CAR(Enum):
   # Toyota
-  ALPHARD_TSS2 = "TOYOTA ALPHARD 2020"
-  AVALON = "TOYOTA AVALON 2016"
-  AVALON_2019 = "TOYOTA AVALON 2019"
-  AVALONH_2019 = "TOYOTA AVALON HYBRID 2019"
-  AVALON_TSS2 = "TOYOTA AVALON 2022"
-  CAMRY = "TOYOTA CAMRY 2018"
-  CAMRYH = "TOYOTA CAMRY HYBRID 2018"
-  CAMRY_TSS2 = "TOYOTA CAMRY 2021"  # TSS 2.5
-  CAMRYH_TSS2 = "TOYOTA CAMRY HYBRID 2021"
-  CHR = "TOYOTA C-HR 2018"
-  CHRH = "TOYOTA C-HR HYBRID 2018"
-  COROLLA = "TOYOTA COROLLA 2017"
-  COROLLA_TSS2 = "TOYOTA COROLLA TSS2 2019"
+  ALPHARD_TSS2 = "Toyota Alphard 2020"
```

Comment:
we need to make sure the value of the enum isn't tied to the description

Reply:
Yeah, going to move to integer enums.

Reply:
Should the values be unique across all makes to make it easy to lookup routes in xx?